operator/pkg/model/translation/envoy_http_connection_manager.go

```diff
@@ -53,6 +53,9 @@ func NewHTTPConnectionManager(name, routeName string, mutationFunc ...HttpConnec
 				},
 			},
 		},
+		InternalAddressConfig: &httpConnectionManagerv3.HttpConnectionManager_InternalAddressConfig{
+			UnixSockets: true,
```

Comment:
can you elaborate why you're setting `UnixSockets` to `true` and not leaving it to its default `false`? Only setting `InternalAddressConfig` should be enough to get rid of the warning.

Reply:
good point :+1:

Reply:
do do we keep `UnixSockets=true` or do we change it to `false`?

Have you checked what's the current behaviour?

Reply:
oh sorry, I was supposed to update to false, iiuc we don't really have unix socket connection for HCM.